tests/test_visitors/test_ast/test_subscripts/test_forbid_consecutive_slices.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    ForbidConsecutiveSlicesViolation,
+)
+from wemake_python_styleguide.visitors.ast.subscripts import (
+    ConsecutiveSlicesVisitor,
+)
+
+no_consecutive = """
```

Comment:
We try to make our test cases as minimal as possible, it means that we can write just:

```python
no_consecutive = 'a[1:]'
```

But, let's keep on to the next comment about parametrization.